worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/QueueHelper.java

```diff
@@ -0,0 +1,26 @@
+package com.fastasyncworldedit.core.queue.implementation;
+
+import com.fastasyncworldedit.core.Fawe;
+import com.fastasyncworldedit.core.queue.IChunkCache;
+import com.fastasyncworldedit.core.queue.IChunkSet;
+import com.fastasyncworldedit.core.queue.implementation.blocks.CharSetBlocks;
+import com.fastasyncworldedit.core.queue.implementation.blocks.IntSetBlocks;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+
+public class QueueHelper {
```

Comment:
This should be a final class as it's a util class, and probably with a private constructor as well (something we should be doing with all util classes tbh)